lib/build.gradle

```diff
@@ -2,6 +2,12 @@ plugins {
     id 'java'
     id 'jacoco'
     id 'com.auth0.gradle.oss-library.java'
+    id 'checkstyle'
+}
+
+checkstyle {
+    toolVersion '10.0'
+    checkstyleTest.enabled = false //We are disabling lint checks for tests
```

Comment:
Why disabling tests? what's the side-effect on enabling there?

Reply:
We are disabling this for tests since there 500+ errors in our test code base as well. I thought we could refactor it in a later release. What would be your suggestion?

Reply:
Makes sense 👍